ckanext/scheming/fanstatic/scheming-repeating-subfields.js

```diff
@@ -17,8 +25,22 @@ this.ckan.module('scheming-repeating-subfields', function (jQuery, _) {
                 $copy.insertAfter($last);
                 // hook for late init when required for rendering polyfills
                 $this.trigger('scheming.subfield-group-init');
+                $count+=1;
                 e.preventDefault();
             });
+
+            $remove.on('click', function(e) {
+                var $groups = $this.find('.scheming-subfield-group'),
+                    $curr = $(this).closest('.scheming-subfield-group'),
+                    $body = $curr.find('.panel-body'),
+                    index = $curr.data('groupIndex'),
+                    field = $curr.data('field');
```

Comment:
what are the index and field variables for?

Reply:
I kept them in incase they are needed for the flash message as well

Reply:
the flash messages would be issued in the controller, not from JS